source/log_channel.c

```diff
@@ -217,10 +230,10 @@ int aws_log_channel_init_background(
     /*
      * Logging thread should need very little stack, but let's defer this to later
      */
-    struct aws_thread_options thread_options = {.stack_size = 0};
+    struct aws_thread_options thread_options = *aws_default_thread_options();
+    thread_options.name = aws_byte_cursor_from_c_str("AwsLogger");
```

Comment:
Background

Reply:
Linux has a 15 char limit on thread names :(
Keep in mind this IS the name of a thread, and only shows up in a debugger when listing threads by name, so it's sort of implicit that it's in the background

Here's me brainstorming some more names
```
123456789012345
AwsBkGrndLogger
AwsBGLogger
```

I'm open to suggestions... bit I still like "AwsLogger" best